### PR TITLE
Fix draft and error alert bugs

### PR DESCRIPTION
**Pull request by ZhaoYangyang0403**
1. Fix the bug of the error alert when submission failed.
2. Upgrade the position of the delete draft button.
image
4. Fix the problem that the name of the deleted draft is still displayed in the selection box.
image
6. Upgrade the way of clearing and deleting a draft.
image
7. Fix plug-in startup error: 'ConfigManager' object has no attribute 'root'.
8. Fix duplicate listeners responding repeatedly when adding description.

### CODE REVIEW

**Comment by yumiguan** [comment]
Fix https://github.com/Meituan-Dianping/lyrebird/issues/762

**Comment by ZhaoYangyang0403** [comment]
Tune clear button:
image

File: frontend/src/components/BugTemplateSelector.vue [hunk 1]
```diff
@@ -71,6 +71,10 @@ export default {
     onDelete () {
       this.$store.dispatch('deleteCache', this.selectedCache)
       this.isShownDeleteModal = false
+    },
```
Comment (yumiguan) [inline frontend/src/components/BugTemplateSelector.vue:74]: delete

File: frontend/src/store/form.js [hunk 2]
```diff
@@ -208,15 +208,16 @@ export default {
       api.createIssue(state.templates[state.selectedTemplateIndex], state.templateDetail,
         state.attachmentsList, state.exportAttachmentList)
         .then(response => {
-          for (let failAttach of response.data.export_fail_attachments) {
-            let extensionName = String(failAttach.split('.').slice(-1))
-            bus.$emit('msg.error', `Add ${failAttach} to attachment error: cannot convert data to a ${extensionName} file.`)
+          if (response.data.code === 1000) {
+            for (let failAttach of response.data.export_fail_attachments) {
+              let extensionName = String(failAttach.split('.').slice(-1))
+              bus.$emit('msg.error', `Add ${failAttach} to attachment error: cannot convert data to a ${extensionName} file.`)
```
Comment (yumiguan) [inline frontend/src/store/form.js:214]: response.data.message

File: frontend/src/store/form.js [hunk 3]
```diff
@@ -363,8 +364,9 @@ export default {
         .then(response => {
           if (response.data.code === 1000) {
             bus.$emit('msg.success', 'Delete success!')
-            commit('setSelectedCache', state.selectedCache)
+            commit('setSelectedCache', '')
```
Comment (yumiguan) [inline frontend/src/store/form.js:367]: commit('setSelectedCache', null)

**Review by yumiguan — APPROVED** [review]

**Pull request merged**